cluster-provision/okd/4.1.7/provision.sh

```diff
@@ -16,12 +16,12 @@ gocli="docker run \
 docker.io/kubevirtci/gocli@${gocli_image_hash}"
 
 ${gocli} provision okd \
---prefix okd-4.1.0 \
+--prefix okd-4.1.7 \
 --dir-scripts ${PARENT_DIR}/okd/scripts \
 --dir-manifests ${PARENT_DIR}/manifests \
 --dir-hacks ${PARENT_DIR}/okd/hacks \
 --master-memory 10240 \
 --installer-pull-token-file ${INSTALLER_PULL_SECRET} \
 --installer-repo-tag release-4.1 \
---installer-release-image quay.io/openshift-release-dev/ocp-release:4.1.0-rc.7 \
+--installer-release-image docker.io/alukiano/ocp-release:4.1.7 \
```

Comment:
can you upload this image into the kubevirtci org ?

Reply:
will do:)